azure-pipelines.release.yml

```diff
@@ -84,7 +84,7 @@ steps:
     displayName: 'Publish Change Requests and Bump Versions'
 
   - script: |
-      yarn ts-node --dir scripts updateReleaseNotes/index.ts --token=$(githubPAT) --apply --debug
+      yarn ts-node scripts/updateReleaseNotes/index.ts --token=$(githubPAT) --apply --debug
```

Comment:
Should be node -r scripts/ts-node-register to ensure the correct tsconfig is used

Reply:
```suggestion
      node -r ./scripts/ts-node-register ./scripts/updateReleaseNotes/index.ts --token=$(githubPAT) --apply --debug
```